invenio_communities/templates/semantic-ui/invenio_communities/details/header.html

```diff
@@ -8,14 +8,21 @@
 #}
 
 {%- from "invenio_theme/macros/truncate.html" import truncate_text %}
+<!-- Image placeholder -->
+<script type="text/javascript">
+  function imagePlaceholder(image) {
+    image.onerror=null;
+    image.src='/static/images/square-placeholder.png';
+  }
+</script>
```

Comment:
As this function seems to be repeated, could we maybe create a macro with it?

Reply:
Yes, see latest comment! :)